ubuntu-20/Dockerfile

```
@@ -1,10 +1,10 @@
-FROM ubuntu:20.04
+FROM ubuntu:focal-20241011
 
 # Label docker image
-LABEL ubuntu_version="20.4"
+LABEL ubuntu_version="ubuntu:focal-20241011"
```

should this label have a colon in it? I would have expected
```suggestion
LABEL ubuntu_version="focal-20241011"
```

---

Done and added makefile